### PR TITLE
add hebrew translation (hebrew reviewers requested)

### PR DESCRIPTION
**Pull request by omriattiya**
## What does this PR do?
Add info | Improve repo

## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [x] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction).
- [x] Used an informative name for this pull request.

### CODE REVIEW

**Comment by eshellman** [comment]
will request hebrew reviewer.

**Comment by ba2sik** [comment]
Hello, I'm a Hebrew speaker and I can review it :)

**Review by 3dw1nM0535 — APPROVED** [review]

File: docs/HOWTO-he.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+[![EbookFoundation/free-programming-books's Monthly Active Contributors graph](https://contributor-overtime-api.apiseven.com/contributors-svg?chart=contributorMonthlyActivity&repo=ebookfoundation/free-programming-books)](https://www.apiseven.com/en/contributor-graph?chart=contributorMonthlyActivity&repo=ebookfoundation/free-programming-books)
+
+</details>
+
+אפילו אם אתה תורם לקוד פתוח שכבר מנוסה, יש דברים שיכולים לשגע אותך. ברגע שהגשת את ה-PR שלך, ***GitHub actions* יריצו את *הלינטר (linter)* שלך, ובדכ ימצאו דברים קטנים על רווחים ושמות באנגלית**. אם יש לך אור ירוק, הכל מוכן ל-review; אבל אם לא, תלחץ על "פרטים" מתחת לבדיקה שנכשלה כדי למצוא למה הלינטר לא עבד, ותתקן את הבעיה עם קומיט חדש ל-branch של ה-PR
+
+בסוף, אם אתה אל בטוח שהמשאב שאתה רוצה להוסיף מתאים בשביל `Free-Programming-Books`, תקרא את הקווים המנחים ב [CONTRIBUTING - תורמים](CONTRIBUTING.md) *זמין גם ב ([תרגומים](README.md#translations) )*
```
Comment (eliranlevi) [inline docs/HOWTO-he.md:35]: "לא" במקום "אל"

File: docs/HOWTO-he.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+* [Creating a pull request](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request)
+* [GitHub Hello World](https://docs.github.com/en/get-started/quickstart/hello-world)
+* [YouTube - GitHub Tutorial For Beginners](https://www.youtube.com/watch?v=0fKg7e37bQE)
+* [YouTube - How To Fork A GitHub Repo and Submit A Pull Request](https://www.youtube.com/watch?v=G1I3HF4YWEw)
+* [YouTube - Markdown Crash Course](https://www.youtube.com/watch?v=HUBNt18RFbo)
+
+אל תהססו לשאול שאלות; כל תורם קוד מתחיל עם ה-PR הראשון שלו. אז... למה שלא תצטרף [לקהילה הגדלה שלנו](https://www.apiseven.com/en/contributor-graph?chart=contributorOverTime&repo=ebookfoundation/free-programming-books)
+
+<details align="center" markdown="1">
+<summary>לחץ כאן כדי לראות את כמות אנשים במהלך הזמן.</summary>
```
Comment (eliranlevi) [inline docs/HOWTO-he.md:25]: כמות האנשים במקום "אנשים"

**Review by eliranlevi — APPROVED** [review]

**Comment by omriattiya** [comment]
@eshellman can you merge it?

**Comment by eshellman** [comment]
Thanks @omriattiya and reviewers, and welcome to free-programming-books!

**Pull request merged**